src/control/server/storage/bdev/backend.go

```diff
@@ -200,28 +200,20 @@ func (b *spdkBackend) Format(pciAddr string) (*storage.NvmeController, error) {
 		return nil, FaultFormatBadPciAddr("")
 	}
 
-	controllers, err := b.Scan()
-	if err != nil {
-		return nil, err
-	}
-
-	foundAddr := false
-	for _, c := range controllers {
-		if c.PciAddr == pciAddr {
-			foundAddr = true
-			break
-		}
+	if !b.binding.initialized {
+		return nil, errors.New("spdk not initialised, please run scan")
```

Comment:
Can you make the error message more explicit so it states the full scan command?  I'm just thinking that if the admin got here in the first place, it might be helpful to be more verbose.

Reply:
done